src/dispatcher.js

```diff
@@ -43,8 +44,23 @@ extend(Dispatcher.prototype, {
     return this._items.length;
   },
 
+
+  addSpy(fn) {
+    this._spies = concat(this._spies, [fn]);
+    return this._spies.length;
+  },
+
+  removeSpy(fn) {
```

Comment:
Could you add a code comment here saying something like "we don't need complicated logic like in remove() method here, because we don't allow adding arbitrary functions as spies, our spies always functions that do only logging"?